<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestHistory</class>
 <widget class="QWidget" name="TestHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_backToChoose">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_currentPage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-family:arial;
font-size:18px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>381</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/stripedBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pageLeft">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/images/pageLeft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_headingDate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_headingResult">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
   </property>
   <property name="text">
    <string> Test Result</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_topBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/topBar.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pageRight">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>590</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/pageRight.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date_1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:arial;
font-size:18px;
color:black;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>lbl_topBar</zorder>
  <zorder>lbl_background</zorder>
  <zorder>lbl_currentPage</zorder>
  <zorder>lbl_result_5</zorder>
  <zorder>btn_pageLeft</zorder>
  <zorder>lbl_result_1</zorder>
  <zorder>lbl_date_6</zorder>
  <zorder>lbl_date_4</zorder>
  <zorder>lbl_result_4</zorder>
  <zorder>lbl_result_2</zorder>
  <zorder>btn_pageRight</zorder>
  <zorder>lbl_result_3</zorder>
  <zorder>lbl_date_2</zorder>
  <zorder>lbl_date_3</zorder>
  <zorder>lbl_date_5</zorder>
  <zorder>lbl_result_6</zorder>
  <zorder>lbl_date_1</zorder>
  <zorder>btn_backToChoose</zorder>
  <zorder>lbl_headingResult</zorder>
  <zorder>lbl_headingDate</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
